### PR TITLE
Edit complete function

### PR DESCRIPTION
**Pull request by yedidya03**
changing complete appointment function to get a value on isNoshow

### CODE REVIEW

**Comment by yaron1m** [comment]
Is this ready for review?

**Comment by yedidya03** [comment]
now it is

File: packages/functions/src/notifications/CompleteAppointmentApi.ts [hunk 1]
```diff
@@ -13,8 +15,16 @@ export const completeAppointmentApi = functions.https.onRequest(
       throw new Error("Invalid appointment id");
     }
 
+    // if (!isNoshow || !(isNoshow in ["true", "false"]) ) {
```
Comment (shwinky) [inline packages/functions/src/notifications/CompleteAppointmentApi.ts:18]: remove this

**Review by shwinky — APPROVED** [review]
approved, just fix the comments

File: packages/common/src/functions-api.ts [hunk 2]
```diff
@@ -83,10 +83,7 @@ export interface CancelAppointmentRequest {
 export const CompleteAppointmentFunctionName = "completeAppointment";
 export interface CompleteAppointmentRequest {
   appointmentId: string;
-}
-
-export interface CompleteAppointmentResponse {
```
Comment (yaron1m) [inline packages/common/src/functions-api.ts:88]: Why was the response deleted?

File: packages/functions/src/notifications/CompleteAppointmentApi.ts [hunk 3]
```diff
@@ -13,8 +15,16 @@ export const completeAppointmentApi = functions.https.onRequest(
...
+    //   throw new Error("Invalid isNoshow value");
+    // }
+
     try {
-      await completeAppointmentFunc(appointmentId, donorId);
+      await completeAppointmentFunc(
+        appointmentId,
+        donorId,
+        isNoshow === "false"
```
Comment (yaron1m) [inline packages/functions/src/notifications/CompleteAppointmentApi.ts:26]: Shouldn't this be `isNoShow === "true"`?

File: packages/functions/src/donor/CompleteAppointmentHandler.test.ts [hunk 4]
```diff
@@ -103,6 +103,40 @@ test("Valid request complete appointment", async () => {
   expect(appointment[0].creatorUserId).toEqual("creatorUserId");
 });
 
+test("Valid request complete appointment with true parameter", async () => {
```
Comment (yaron1m) [inline packages/functions/src/donor/CompleteAppointmentHandler.test.ts:106]: 2 Small comments:
1. The name of the test is not explanatory enough. What does the true or false mean? Consider something like "With No Show".
2. The two tests you added are identical, except for the status. For tests like this you can use `test.each`. See an example here:
https://github.com/startach/ZM-BloodComponents/blob/fa2be91d73aa46eac0e2a4db6976aafd464ef44b/packages/functions/src/coordinator/DeleteAppointmentsHandler.test.ts#L101-L134

**Review by yaron1m — COMMENTED** [review]
Please make sure you also address my comments before merging

**Pull request merged**